openedx/core/djangoapps/content_tagging/tests/test_rules.py

```diff
@@ -534,10 +534,10 @@ def test_object_tag_disabled_taxonomy(self, perm, tag_attr):
     )
     @ddt.unpack
     def test_object_tag_no_orgs(self, perm, tag_attr):
-        """Only staff & superusers can create/edit an ObjectTag with a no-org Taxonomy"""
+        """Only superusers can create/edit an ObjectTag with a no-org Taxonomy"""
         object_tag = getattr(self, tag_attr)
         assert self.superuser.has_perm(perm, object_tag)
-        assert self.staff.has_perm(perm, object_tag)
+        assert not self.staff.has_perm(perm, object_tag)
```

Comment:
This is a side effect of the change. When we didn't check cross-tagging, a staff user could change objecttags from taxonomies without orgs.

Reply:
The issue is that it is not trivial to differentiate a no-org taxonomy and a taxonomy from another org.

I fixed here: 2c7e806133d9b394fa11bfc8b0073ff41796a51c

But now taxonomy admins can also use a Taxonomy from orgA to tag an object from orgB (like superuser).